src/jit/importer.cpp

```diff
@@ -19604,7 +19929,7 @@ GenTree* Compiler::impInlineFetchArg(unsigned lclNum, InlArgInfo* inlArgInfo, In
             assert(!argInfo.argIsUsed);
 
             /* Reserve a temp for the expression.
-            * Use a large size node as we may change it later */
+             * Use a large size node as we may change it later */
```

Comment:
nit: if you're going to change this comment, you should just change it to a C++ style comment (`\\`).